### DISCUSSION TITLE
[TCOTD2] Should stop editing the map by the 30th

### DISCUSSION THREAD

**Discussion by nashmuhandes** [Realm667/Re-Releases#288] [General]
Should feature freeze the entire project. Can't risk any more breakages this close to release (especially from changing the map .wad) and this mod takes a very long time to test (1 hour+ if playing "normally", even 20 - 30 minutes with the debug mode enabled)

I am generally done with this mod and would rather not sink another 30 minutes - 1  hour of my life to replay this thing all over again :(

Attention @Tormentor667

**Comment by Tormentor667** [comment]
I am done with my map changes now after my third run. Nothing I could find anymore that is worth being edited :)

**Comment by nashmuhandes** [comment]
Okay. I will be playing it ... again... for one final time... X) (to check for breakages)
